package.json

```diff
@@ -1802,15 +1802,18 @@
                     "type": "boolean",
                     "default": false,
                     "description": "Experimental feature to enable execution analysis in notebooks",
-                    "scope": "application"
+                    "scope": "application",
+                    "tags": [
+                        "experimental"
+                    ]
                 },
                 "jupyter.enableKernelCompletions": {
                     "type": "boolean",
                     "default": true,
                     "markdownDescription": "%jupyter.configuration.jupyter.enableKernelCompletions.markdownDescription%",
                     "scope": "application",
                     "tags": [
-                        "experimental"
+                        "onExP"
```

Comment:
This setting looks like it already rolled out to 100% of users. Could we stop the experiment and consider this setting a stable setting?

Reply:
Yes, thank you.